internal/server/v0/triple/golden/get_triples/limit.json

```diff
@@ -110,7 +110,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/sm3m2w3"
```

Comment:
@pradh should we expect the provenance be stable?

Reply:
Yes, but see my comment above